drop.py

```diff
@@ -5,9 +5,6 @@
 c = Configuration("config.json")
 client = MongoClient(c.mongo_uri())
```

Comment:
Drop `()` as `c.mongo_uri` is now a property, and can't be called as a function.